public/index.html

```diff
@@ -1,22 +1,18 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" dir="ltr" >
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content=""
-    />
-
+    <meta name="description" content="An app built with every body and heart in mind" />
+    <meta name="Angela Battillo" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-
+    <link rel="stylesheet" media="screen" href="https://fontlibrary.org//face/opendyslexic" type="text/css"/>
     <title>Head To Toe</title>
   </head>
   <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <noscript></noscript>
```

Comment:
If we're removing the text, are these tag necessary?

Reply:
We will need to implement the appropriate alt tag.